packages/x-scheduler/src/joy/agenda-view/AgendaView.css

```diff
@@ -0,0 +1,70 @@
+.AgendaViewContainer {
+  width: 100%;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  border-radius: var(--radius-4);
+  overflow-y: scroll;
+}
+
+.AgendaViewRow {
+  display: flex;
+  border-bottom: 1px solid var(--border-color);
+}
+.AgendaViewRow:last-child {
+  display: flex;
+  border-bottom: none;
+}
+
+.DayHeaderCell {
+  display: flex;
+  align-items: flex-start;
+  border-right: 1px solid var(--border-color);
+  padding: var(--space-4);
+  color: var(--gray-9);
+  gap: var(--space-1);
+}
+
+.DayHeaderCell.Today {
+  background-color: var(--primary-2);
+  color: var(--primary-13);
+}
+
+.DayNumberCell {
+  font-size: var(--font-size-5);
+  font-weight: var(--font-weight-2);
+  line-height: 1;
+  min-width: 4ch;
+  text-align: center;
+}
+
+.WeekDayCell {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-2);
+}
+
+.WeekDayName {
+  font-size: var(--font-size-2);
+  font-weight: var(--font-weight-3);
+  line-height: 1;
+  /* TODO find a more robust way to make the header cells have the same width */
+  min-width: 80px;
+}
+
+.YearAndMonth {
+  font-size: var(--font-size-1);
+  color: var(--gray-8);
+  line-height: 1;
+}
+
+.Today .YearAndMonth {
```

Comment:
Don't we need CSS collision between our various components if we don't use prefixed classes?